Additional proofreading

cgreene opened the pull request:
I did this on top of @agitter's branch. 🤞 that making this against the master repo works now!

agitter committed:
Update versioned manuscript link

agitter committed:
Reduce redundancy with closing paragraph

agitter committed:
Intro edits

agitter committed:
Discussion edits

agitter committed:
Move living document paragraph again

agitter committed:
Edit discussion

cgreene committed:
minor changes, flow & such

cgreene committed:
Merge branch 'master' into agitter-proofreading

cgreene committed:
remove merge-induced newline

cgreene commented on content/02.main-text.md:
```diff
@@ -2,12 +2,12 @@
-However, the scientific community requires tools whose workflows encourage openness rather than stifle it.
-For example, manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, where anyone is able to contribute and the contribution history is preserved and public.
-Hence, we report a new tool and workflow for authoring scholarly manuscripts in the open, as well as the collaborative project that led to its creation.
+However, the scientific community requires tools whose workflows encourage openness.
```
I saw a @dhimmel question on this in the PR into @agitter's repo. It is unclear that it's fair to describe current tools as stifling research. It seems more likely that they were simply designed without that use case in mind.

agitter replied:
@dhimmel wrote https://github.com/agitter/meta-review/pull/1#discussion_r206365062
> Is the point still clear without "rather than stifle it"?

I think the point is still sufficiently clear with the new phrasing.

cgreene commented on content/02.main-text.md:
```diff
@@ -47,23 +47,23 @@ Although they work well for editing text, they lack sufficient features for mana
 <sup>2</sup>Conversations about modifications take place on the document as comments, annotations, or unsaved chats.
-There is no separate integrated forum for discussing and editing revisions.
-<sup>3</sup>Simultaneous edits online can be attributed to a single author in a git commit.
+There is no integrated forum for discussing and editing revisions.
```
I found that having both `separate` and `integrated` was confusing. Perhaps we need to adjust the phrasing of this and/or the sentence before to enhance clarity.

cgreene commented on content/02.main-text.md:
```diff
@@ -47,23 +47,23 @@ Although they work well for editing text, they lack sufficient features for mana
-There is no separate integrated forum for discussing and editing revisions.
-<sup>3</sup>Simultaneous edits online can be attributed to a single author in a git commit.
+There is no integrated forum for discussing and editing revisions.
+<sup>3</sup>In some circumstances, a git commit containing different edits made at the same time by distinct authors may be attributed to a single author.
```
I struggled with how to word this. But the essence is that authorship gets assigned to the person who makes the commits, not the person who makes the edits.

agitter replied:
@dhimmel wrote https://github.com/agitter/meta-review/pull/1#discussion_r206364776
> Isn't the Overleaf issue worse? Are we sure any commits are correctly attributed to their author?

> I think it should be more like, "git commits created by Overleaf do not correspond to modular revisions and do not retain accurate authorship information."

The Overleaf documentation I copied in https://github.com/greenelab/meta-review/issues/27#issuecomment-407889342 details how they use git commits (in the v1 interface).  The commit author is complicated.  I believe it depends on whether the last editor before a commit is triggered is logged in or not.  The edits do not actually have to be made at the same time to confound attribution.  They can be combined in a single commit if they are made before a commit-generating event.

Proposed phrasing:
> In some circumstances, Overleaf git commits are not modular.  Edits made by distinct authors may be attributed to a single author.

cgreene commented on content/02.main-text.md:
```diff
@@ -2,12 +2,12 @@
 
 Open scholarly writing, a form of crowdsourcing [@pmcid:PMC4719068], has particular benefits for review articles, which present the state of the art in a scientific field [@doi:10.1371/journal.pcbi.1003149].
-Literature reviews are typically written in private by a closed team of colleagues.
+Literature reviews are typically written in private by an invited team of colleagues.
```
I'm not sure they are actively `closed` so much as passively closed. Thus why I switched the phrasing.

agitter commented on content/02.main-text.md:
```diff
@@ -24,10 +24,10 @@ In total, the Deep Review attracted {{deep_review_authors}} authors who were not
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
-To address these challenges, we developed a manuscript writing process using the Markdown language, the GitHub platform, and our new Manubot tool for automating manuscript generation.
+To address these challenges, we developed a manuscript writing process using the Markdown language, the GitHub platform, and our new [Manubot tool](https://github.com/greenelab/manubot) for automating manuscript generation.
```
You can remove the link here. I moved it elsewhere in #92 (to `Manubot is a system for writing scholarly manuscripts via GitHub that is built upon our Python package of the same name.`).

agitter reviewed (changes requested):
LGTM except for a few minor comments.

cgreene committed:
use suggested wording re: overleaf

from @agitter https://github.com/greenelab/meta-review/pull/93#discussion_r206494808

cgreene committed:
remove manubot link

cgreene commented:
I think I addressed them @agitter . If it looks good, feel free to approve. I will wait for @dhimmel before merging.

agitter commented:
@cgreene before we merge this, can you please add an annotation to the collaborative writing table that we are reporting on Overleaf v1?   Or say `Overleaf v1` in the Overleaf column header?

We've started testing Overleaf v2, and it may have a different set of features.  In particular, they have a different system for GitHub integration that could possibly affect the git commit behavior.

cgreene committed:
note that overleaf is version 1
